steps/cli/oc.go

```diff
@@ -19,10 +19,14 @@ var _ = gauge.Step("Create <table>", func(table *m.Table) {
 	}
 })
 
-var _ = gauge.Step("Apply <table>", func(table *m.Table) {
+var _ = gauge.Step("Apply in <ns> namespace <table>", func(ns string, table *m.Table) {
```

Comment:
1. duplicate step
2. we have `Validate default auto prune cronjob in target namespace` so name the new one `Apply <table> in namespace <ns>`